shell/composables/focusTrap.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * focusTrap is a composable based on the "focus-trap" package that allows us to implement focus traps
+ * on components for keyboard navigation is a safe and reusable way
+ */
+import { watch, nextTick, onMounted, onBeforeUnmount } from 'vue';
+import { createFocusTrap, FocusTrap } from 'focus-trap';
+
+export function basicSetupFocusTrap(containerSelector: string) {
+  let focusTrapInstance = {} as FocusTrap;
```

Comment:
I think that we can just specify the type:

```suggestion
  let focusTrapInstance: FocusTrap;
```